test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -0,0 +1,128 @@
+package kubernetes
```

Comment:
I would leave that up to @seanknox, @sgoings, and @anhowe to figure out. That was not what this PR was about.